README.md

```diff
@@ -1,88 +1,55 @@
-<a name="readme-top"></a>
+# Book Motorcycles App
```

Comment:
# Backend review

- [ ] On the License section of your readme:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/abee10dd-577a-4ea6-b394-f7477ff6f1f5)

The link does not work because it is pointing to a file that does not exist.

Kindly, fix the link to the correct license file that exists on your project.


-[ ] Kindly, remove this note.  This now was meant just for you:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/5f1eb1ea-ac21-483b-8a7b-c9112516695d)


- [ ] These sections of your readme are not properly filled:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/51dae7b4-d304-4ab2-b746-0bc2154e97ff)

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/51b19914-0fd9-4b79-bda9-f263d82ef436)


Kindly fill them out with info about your project.

- [ ] This is the default test that comes with React:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/0e1a4bbe-899a-4b8a-a5bc-289f39a5f0ef)

And its failing:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/624d995d-5bd5-4d6f-8512-97eab1002e9f)

Since you are not including frontend tests, kindly remove the file to prevent confusion. 👍

- [ ] IMPORTANT: When I run the project, this happens:

1. Webpack fails to compile

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/e8a4addb-34a8-4eac-bf6e-c2c9147b8e2a)

## Explanation

All these imports have a typo in the "Components" folder:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/8781eccf-6d98-4936-8d30-8bce5a82ff97)

The import is capitalized, the folder is not.

Kindly, correct these imports so the app is able to run correctly.

_note:  This means I wasn't able to run the frontend app and test its styling and functionality.  This will be taken a look in the next review. 👍_ 


- [ ] Kindly, refactor these functions to ES6:
 
![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/e7ae4fd4-c76b-415c-9260-a83808e789c0)

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪